home.html

```diff
@@ -68,42 +72,253 @@
 
   <div class="container text-center mt-4">
     <button class="btn btn-success" data-bs-toggle="modal" data-bs-target="#addCarModal">Add Car</button>
-  </div>
 
+    <div class="card-body">
+        <table id="myTable" class="table table-border table-striped">
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>Name</th>
+              <th>Model</th>
+              <th>Year</th>
+              <th>Type</th>
+              <th>Action</th>
+            </tr>
+          </thead>
+          <tbody>
+          </tbody>
+        </table>
+      </div>
+      </div>
+      
+      <script>
+      $(document).ready(function() {
+        // Load initial data
+        loadData();
+      
+        // Add button click event
+        $('#addButton').click(function() {
+          var name = $('#nameInput').val();
+          var model = $('#modelInput').val();
+          var year = $('#yearInput').val();
+          var type = $('#typeInput').val();
+      
+          $.ajax({
+            type: 'POST',
+            url: '/staging/cars',
```

Comment:
You should use: 
```suggestion
            url: 'rest/staging/cars/user/{userId} ',
```
to get cars for the current user.